editor/editor_inspector.cpp

```diff
@@ -1824,8 +1824,15 @@ void EditorInspectorArray::_move_element(int p_element_index, int p_to_pos) {
 				properties_as_array.remove_at(p_element_index);
 			} else {
 				// Move the element.
-				properties_as_array.insert(p_to_pos, properties_as_array[p_element_index].duplicate());
-				properties_as_array.remove_at(p_to_pos < p_element_index ? p_element_index + 1 : p_element_index);
+				if (p_to_pos - p_element_index == 1) {
+					properties_as_array.insert(p_to_pos + 1, properties_as_array[p_element_index].duplicate());
+					properties_as_array.remove_at(p_element_index);
+				}
+				else {
```

Comment:
```suggestion
				} else {
```

Reply:
Thanks, yes, there is some clang format correction to do. I'm on it.